src/pup/src/lib.rs

```diff
@@ -106,11 +106,14 @@ impl Pup {
         };
 
         // Create exFAT reader.
-        let _fat = match ExFat::open(entry) {
+        let fat = match ExFat::open(entry) {
             Ok(v) => v,
             Err(e) => return Err(DumpSystemImageError::CreateImageReaderFailed(e)),
         };
 
+        // Dump files.
+        for item in fat {}
+
```

Comment:
Was there supposed to be something implemented in this loop?

Reply:
Right. We need a complete implementation of `Directory` and `File` from `exfat` before we can implement this loop.